frontend/src/app/Settings/Users/ManageUsers.tsx

```diff
@@ -63,7 +84,19 @@ const ManageUsers: React.FC<Props> = ({
   deleteUser,
   getUsers,
 }) => {
-  const [activeUser, updateActiveUser] = useState<SettingsUser>();
+  const [activeUser, updateActiveUser] = useState<LimitedUser>();
+  const [
+    userWithPermissions,
+    updateUserWithPermissions,
+  ] = useState<SettingsUser>();
+  const [queryUserWithPermissions] = useLazyQuery<SingleUserData, {}>(
```

Comment:
The rest of the queries and mutations used here are defined in `ManageUsersContainer` and passed through as props (and, I think, use `useQuery`).  If the other way is the convention then it would be good to follow, but if not then ignore this.

Reply:
I think it is likely convention, but it's unfortunately not possible to replicate with this query because this one has a dynamic query parameter. To bubble it back to `ManageUsersContainer` we'd need to move the `activeUser` state there as well, which I would rather avoid since the container doesn't manage that state at all.